### DISCUSSION TITLE
Installed LME, but can't access GUI anymore

### DISCUSSION THREAD

**Discussion by jp18018** [cisagov/LME#71] [FAQ]
Recently setup LME and was able to login to the GUI.  However, it does not bring up the web interface any more.
How can I troubleshoot?

**Comment by llwaterhouse** [comment]
Try this on your Linux server:
`sudo chown -R 1000:1000 /opt/lme/backups` 
and see if the docker.elastic.co/elasticsearch/elasticsearch:8.7.1 container stays running (`docker stack ps lme`)

**Comment by llwaterhouse** [comment]
If you have further questions about this, please submit a gitHub Issue, https://github.com/cisagov/LME/issues.

Thank you.
